### PR TITLE
Chat: bigger error when user deleted

### PR DESCRIPTION
**Pull request by mlsteele**
Used to just say "Deleted"

image

### CODE REVIEW

File: go/chat/boxer.go [hunk 1]
```diff
@@ -89,38 +89,45 @@ func (b *Boxer) makeErrorMessage(msg chat1.MessageBoxed, err UnboxingError) chat
...
+			}
+			return NewTransientUnboxingError(err)
 		}
 	}
 
-	switch err.(type) {
-	case libkb.DeletedError:
-		return true
+	switch err := err.(type) {
+	case libkb.UserDeletedError:
```
Comment (mmaxim) [inline go/chat/boxer.go]: Does `DeletedError` not happen anymoer?

Comment (mlsteele) [reply go/chat/boxer.go]: I renamed it. It was only being used for when a user is deleted. And I thought maybe the name was part of why the messages were so vague.

Also I'm working on making the gui message better cause it's still bad.

Comment (mmaxim) [reply go/chat/boxer.go]: Ah ok, I should have scrolled down further.

**Comment by mlsteele** [comment]
The error munging is complicated enough. I'm inclined to leave it at this.

**Pull request merged**